tests/conftest.py

```diff
@@ -488,13 +490,32 @@ def running_clients(hlwm, running_clients_num):
     return hlwm.create_clients(running_clients_num)
 
 
-@pytest.fixture()
-def x11():
-    from Xlib import X, display, Xutil, Xatom
+@pytest.fixture(scope="session")
+def x11_connection():
+    """ Long-lived fixture that maintains an open connection to the X11 display
+    for the entire duration of all tests. This avoids issues caused by Xvfb and
+    Xephyr resetting all properties whenever the last connection is closed. It
+    is probably a bit more efficient, too. """
+    class X11Connection:
+        def __init__(self):
+            self.display = Xlib.display.Display()
+
+        def shutdown(self):
+            self.display.close()
+
+    x11_connection_ = X11Connection()
+    yield x11_connection_
+    x11_connection_.shutdown()
```

Comment:
Here, we don't really need the class, right? We could just define:
```
display = Xlib.display.Display()
yield display
display.close()
```
But we can also keep the class for the case that we want to put more attributes here.

Reply:
Oh, you're totally right! I was quite stuck in the pattern it seems... I'll change it